config/piwik.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration:
+# 
+# disabled
+#   false if tracking tag should be shown
+# use_async
+#   Set to true if you want to use asynchronous tracking
+# url
+#   The url of your piwik instance (e.g. localhost/piwik/
+# id_site
```

Comment:
me forgetting to clean the mess left by piwik gem ;)

By the way, would you prefer using the piwik gem or the simple solution implemented at the moment ?